addons/crm/views/crm_lead_views.xml

```diff
@@ -208,7 +208,7 @@
                             </group>
                             <group>
                                 <field name="user_id" domain="[('share', '=', False)]"
-                                    context="{'default_groups_ref': ['base.group_user', 'base.group_partner_manager', 'sales_team.group_sale_salesman_all_leads'], 'team_id': team_id}"/>
```

Comment:
Oooh remaining of our famous "create users with some default groups" ... snif. Good ol' openerp.